corehq/apps/app_manager/templates/app_manager/partials/modules/module_view_settings.html

```diff
@@ -51,6 +51,29 @@ <h4 class="panel-title panel-title-nolink">{% trans "Menu" %}</h4>
               {% include "app_manager/partials/modules/module_view_case_type.html" %}
             {% endif %}
 
+            {% if request|toggle_enabled:'USH_CASE_CLAIM_UPDATES' %}
+            <div class="form-group">
+              <label class="{% css_label_class %} control-label">
+                 {% trans "Case List and Case Search Types" %}
+                <span class="hq-help-template"
+                      data-title="{% trans "Case List and Case Search Types" %}"
+                      data-content="{% blocktrans %}
+                             Cases of these types will be displayed in Case List and Case Search screens.
+                             <a href='https://confluence.dimagi.com/display/ccinternal/Case+Search+and+Claim'>(More Information)</span>
+                             {% endblocktrans %}">
+                </span>
+              </label>
+              <div class="{% css_field_class %}">
+                <select name='additional_case_types' class='form-control multiselect-caselist' multiple="multiple" >
+                  {% for case_type in case_types %}
+                    <option value="{{ case_type }}" {% if case_type in  module.additional_case_types %}selected{% endif %}>{{ case_type }}</option>
+                  {% endfor %}
+                </select>
+                <input name="detail_type" type="hidden" value="{{ detail_type }}" />
```

Comment:
What's this? Doesn't look like it's referenced.

Reply:
Removed!